### ISSUE TITLE
Error trying to use docker run command in README

### ISSUE THREAD

**Issue by cjdeleo** [domnikl/schema-registry-gitops#29]
```
> docker run -v "$(pwd)/examples":/data domnikl/schema-registry-gitops plan --properties /data/client.properties /data/schema-registry.yml

2022-08-09 19:36:27.563 INFO Starting MainKt using Java 17-ea on 4263ce7d318d with PID 1 (/home/cuser/schema-registry-gitops.jar started by root in /home/cuser)
2022-08-09 19:36:27.574 INFO No active profile set, falling back to 1 default profile: "default"
2022-08-09 19:36:30.668 INFO Started MainKt in 3.605 seconds (JVM running for 4.408)
2022-08-09 19:36:30.947 WARN Unable to get bean of class class dev.domnikl.schema_registry_gitops.cli.Plan, using default Picocli factory
picocli.CommandLine$InitializationException: Cannot instantiate dev.domnikl.schema_registry_gitops.cli.Plan: the class has no constructor
	at picocli.CommandLine$DefaultFactory.create(CommandLine.java:5543)
	at picocli.CommandLine$Model$CommandUserObject.getInstance(CommandLine.java:11918)
	at picocli.CommandLine$Model$CommandSpec.userObject(CommandLine.java:6285)
	at picocli.CommandLine$Interpreter.clear(CommandLine.java:13111)
	at picocli.CommandLine$Interpreter.parse(CommandLine.java:13156)
	at picocli.CommandLine$Interpreter.processSubcommand(CommandLine.java:13468)
	at picocli.CommandLine$Interpreter.processArguments(CommandLine.java:13374)
	at picocli.CommandLine$Interpreter.parse(CommandLine.java:13177)
	at picocli.CommandLine$Interpreter.parse(CommandLine.java:13146)
	at picocli.CommandLine$Interpreter.parse(CommandLine.java:13047)
	at picocli.CommandLine.parseArgs(CommandLine.java:1478)
	at picocli.CommandLine.execute(CommandLine.java:2077)
	at dev.domnikl.schema_registry_gitops.SchemaRegistryGitopsApplication.run(Main.kt:33)
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:771)
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:755)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:164)
	at dev.domnikl.schema_registry_gitops.MainKt.main(Main.kt:61)
Caused by: java.lang.NoSuchMethodException: dev.domnikl.schema_registry_gitops.cli.Plan.<init>()
	at java.base/java.lang.Class.getConstructor0(Class.java:3530)
	at java.base/java.lang.Class.getDeclaredConstructor(Class.java:2700)
	at picocli.CommandLine$DefaultFactory.create(CommandLine.java:5514)
	at picocli.spring.PicocliSpringFactory.create(PicocliSpringFactory.java:26)
	at picocli.CommandLine$DefaultFactory.create(CommandLine.java:5541)
```
Any suggestion on what im doing wrong?

**Comment by cjdeleo** [comment]
I get no issues using 1.3.0 but latest doesn't work

**Comment by domnikl** [comment]
Fixed it in 1.6.0, which is now latest (after you `docker pull` again).
